app/models/host_aggregate.rb

```diff
@@ -18,6 +18,7 @@ class HostAggregate < ApplicationRecord
   has_many   :vim_performance_states, :as => :resource
 
   virtual_total :total_vms, :vms
+  virtual_column :availability_zone, :type => :integer
```

Comment:
I think this is a string (effectively the ems_ref of the AZ), @andyvesel  ?

Reply:
I see `availability_zone` defined as a hash key in inventory and as a belongs_to, but now where do I see this defined as an attribute.

what are we trying to expose here?
If we are delegating this to another class, then a virtual delegate may be right.
If this is a delegation of a relationships, I think active record likes alias_attribute (not thrilled with their naming but...)

maybe we want `availability_zone_id`?

Reply:
@kbrock there is a `#metadata` text attribute that has an `"availability_zone"` key, we need the value from that key.  The Openstack subclass of this exposes an accessor already maybe we need to pull that up into the base class

Reply:
@agrare good one. Yes, if we want to add this method here, we need to define the ruby method as well.

I am not sure if this concept means anything to others, but if we want attributes to be available, we tend to need them to be defined at the base layer